cmake/Dependencies.cmake

```diff
@@ -539,6 +539,12 @@ if(USE_XNNPACK AND NOT USE_SYSTEM_XNNPACK)
     set(__caffe2_CMAKE_POSITION_INDEPENDENT_CODE_FLAG ${CMAKE_POSITION_INDEPENDENT_CODE})
     set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
+    if(Win32)
+      # Windows MSVC is not have libm, and this option possible to bring in linker issue:
```

Comment:
"Disable libm dependency explicitly to avoid symbol conflict for XNNPACK as Windows runtime has provided the math functions - https://github.com/pytorch/pytorch/issues/134989"